### DISCUSSION TITLE
Feature Request: Show number of participants in room

### DISCUSSION THREAD

**Discussion by DeveloperWil** [venueless/venueless#339] [Ideas]
Would be great to have the number of participants in a room shown in the UI.

Lots of speakers ask how many were watching.  There's plenty of room in the UI to display this.

**Comment by rashfael** [comment]
For numbers in the sidebar there already is https://github.com/venueless/venueless/issues/179
Showing a attendee count in the current room is something a bit different though.
Organizers have access to these numbers through our reports.
